src/test/java/hudson/plugins/jobConfigHistory/ConfigurationAsCodeTest.java

```diff
@@ -0,0 +1,68 @@
+package hudson.plugins.jobConfigHistory;
+
+import io.jenkins.plugins.casc.ConfigurationContext;
+import io.jenkins.plugins.casc.ConfiguratorRegistry;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import io.jenkins.plugins.casc.model.CNode;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+
+import static io.jenkins.plugins.casc.misc.Util.getUnclassifiedRoot;
+import static io.jenkins.plugins.casc.misc.Util.toStringFromYamlFile;
+import static io.jenkins.plugins.casc.misc.Util.toYamlString;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ConfigurationAsCodeTest {
+    
+    @Rule
+    public JenkinsConfiguredWithCodeRule r = new JenkinsConfiguredWithCodeRule();
+
+    @Issue("JENKINS-55667")
+    @Test
+    @ConfiguredWithCode("configuration-as-code.yml")
+    public void should_support_configuration_as_code() {
+        JobConfigHistory plugin = PluginUtils.getPlugin();
+        assertEquals("/var/jenkins_home", plugin.getHistoryRootDir());
+        assertEquals("30", plugin.getMaxDaysToKeepEntries());
+        assertEquals("20", plugin.getMaxEntriesPerPage());
+        assertEquals("10000", plugin.getMaxHistoryEntries());
+        assertEquals("test1\\.xml|test2\\.xml", plugin.getExcludePattern());
+        assertEquals("SYSTEM,user1,user2", plugin.getExcludedUsers());
+        assertEquals("never", plugin.getShowBuildBadges());
+        assertTrue(!plugin.getShowChangeReasonCommentWindow());
+        assertTrue(!plugin.getSkipDuplicateHistory());
+        assertTrue(!plugin.getSaveModuleConfiguration());
```

Comment:
Wouldn't it be easier to use `assertFalse` instead of negation and `assertTrue`?